src/containers/layout/app-breadcrumbs/AppBreadCrumbs.styles.js

```diff
@@ -26,6 +26,9 @@ export const styles = {
       flexWrap: 'nowrap',
       overflowX: 'auto',
       '&::-webkit-scrollbar': { display: 'none' }
+    },
+    a: {
+      display: 'block'
```

Comment:
Not clearly why style for `a` is working) Last element is link, maybe you should write (change) styles for `link`?